pkg/user/UserService.go

```diff
@@ -975,3 +976,16 @@ func (impl UserServiceImpl) IsSuperAdmin(userId int) (bool, error) {
 	}
 	return isSuperAdmin, nil
 }
+
+func (impl UserServiceImpl) GetByIdIncludeDeleted(id int32) (*bean.UserInfo, error) {
```

Comment:
This method is giving only userinfo. 
while getById gives roles and other data as well along with userInfo. 

I hope this is intentional and would not have any impact

Reply:
yes this was intentional, we required only email for the service.

Reply:
ok